fs/mount/fs_automount.c

```diff
@@ -813,7 +813,7 @@ FAR void *automount_initialize(FAR const struct automount_lower_s *lower)
   FAR char *devpath = lib_get_pathbuffer();
   if (devpath == NULL)
     {
-      return;
+      return NULL;
```

Comment:
memory leak on line 828, please check the lifecycle of path buffer in all spaces for this feature